service/src/main/resources/application-development.properties

```diff
@@ -1,5 +1,5 @@
 service.environment=development
-service.manager_key=${MANAGER_KEY}
+service.manager_key=${MANAGER_KEY:pond rebuild kick kitten taxi region burst people sadness man news young evil lemon decrease vault always daring dignity either van mandate celery taste}
```

Comment:
decided to just stick this in here because its bank3 which is handy given it integrates nicely with smart contract docs